### PR TITLE
Pact provider tests added

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

**Comment by spattisson** [comment]
I've not run this yet but it looks good to me

File: server/services/publish-message-action.js [hunk 1]
```diff
@@ -6,14 +6,30 @@ const scheduleSender = new MessageSender(config.scheduleQueueConfig, config.sche
...
   } catch (err) {
     console.log(err)
     throw err
   }
 }
 
-module.exports = { publishMessageAction }
+function getCalculationMessage (claim) {
+  return claim
+}
```
Comment (jaucourt) [inline server/services/publish-message-action.js:23]: I appreciate this is improving consistency, but it feels a little like premature optimisation. I can see it's needed for the schedule message, but it seems unnecessary for the calculation message

Comment (johnwatson484) [reply server/services/publish-message-action.js:23]: Agreed but I included an explanation of why this is needed for Pact in Confluence.

_A prerequisite of the Pact message provider being able to test from the provider side, is that the application must have a function that outputs the message content so Pact can validate it against a contract.

In the claim service, this was not the case as the message content is received via an Http post and passed directly to SQS.  To support contract testing, the code was rewritten as below.  This does solve some outstanding technical debt where the full claim was unnecessarily sent to the schedule queue when only the claimId was needed but does introduce extra complexity to the calculation queue logic that delivers no application benefit._

Comment (jaucourt) [reply server/services/publish-message-action.js:23]: That sounds good, and making code more testable is never a bad thing.

**Pull request merged**